tests/unit/checkpoint/test_latest_checkpoint.py

```diff
@@ -0,0 +1,62 @@
+import deepspeed
+
+from tests.unit.common import DistributedTest
+from tests.unit.simple_model import *
+
+from tests.unit.checkpoint.common import checkpoint_correctness_verification
+
+
+class TestLatestCheckpoint(DistributedTest):
+    world_size = 1
+
+    def test_existing_latest(self, tmpdir):
+        config_dict = {
+            "train_batch_size": 2,
+            "steps_per_print": 1,
+            "optimizer": {
+                "type": "Adam",
+                "params": {
+                    "lr": 0.00015
+                }
+            }
+        }
+        hidden_dim = 10
+        args = args_from_dict(tmpdir, config_dict)
```

Comment:
not super important, but I've been trying to remove unnecessary code. This args_from_dict is not necessary anymore and you can replace all `deepspeed.initialize(args=args, ...` with `deepspeed.initialize(config=config_dict, ...`

Reply:
Done